internal/controller/operator/factory/vmcluster/vmcluster.go

```diff
@@ -1486,6 +1486,10 @@ func buildVMauthLBDeployment(cr *vmv1beta1.VMCluster) (*appsv1.Deployment, error
 	args = cr.Spec.License.MaybeAddToArgs(args, vmv1beta1.SecretsDir)
 
 	args = build.AddExtraArgsOverrideDefaults(args, spec.ExtraArgs, "-")
+
+	volumes, vmounts = spec.License.MaybeAddToVolumes(volumes, vmounts, vmv1beta1.SecretsDir)
```

Comment:
As far as I see, it should be supported already 	volumes, vmounts = cr.Spec.License.MaybeAddToVolumes(volumes, vmounts, vmv1beta1.SecretsDir)